modules/reliability/templates/menu_reliability.tpl

```diff
@@ -99,7 +99,7 @@
                 <span class="glyphicon glyphicon-chevron-up pull-right" style="display:none" id="swapUp"></span>
             </div>
             <div class="panel-body" style="display:none" id="swap-body">
-                <form method="post" action="main.php?test_name=reliability">
+                <form method="post" action="?reliability">
```

Comment:
Yes, you are right. I have changed this URL and it gets an error.
"You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use ..." 
Could you give the advice to fix this issue?